src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table-key-value/table-key-value.component.ts

```diff
@@ -42,6 +42,42 @@ export class TableKeyValueComponent implements OnInit {
   constructor() { }
 
   ngOnInit() {
+    if (!this.data) {
+      throw new Error('No data provided');
+    }
+    if (this.data.hasOwnProperty('length')) {
```

Comment:
Could you replace this and the following conditions with lodash validations?

Reply:
done